### DISCUSSION TITLE
Project card images not rendering in mobile device

### DISCUSSION THREAD

**Discussion by manandey** [alshedivat/al-folio#562] [Q&A]
Hi,

The project card images I have used are rendering properly when I open the site in my laptop's chrome browser but are not rendering when I open the same on my mobile device. URL for reference: [Projects](https://manandey.github.io/projects/ )

It would be great if someone can suggest any fix.
cc. (@alshedivat)

Thanks,
Manan

**Comment by alshedivat** [comment]
thanks for bringing this up. I think this is related to #537 issue. on the page you linked, if I zoom in, images start disappearing. this is likely due to a small bug in how responsive image sizes are generated.
